verify/verify-flags/known-flags.txt

```diff
@@ -24,3 +24,4 @@ ping-server-address
 ping-server-bind-address
 ping-sleep-duration
 master-internal-ip
+enable-violations-api-responsiveness-prometheus
```

Comment:
NIT, can be done in other PR, could you sort the entries alphabetically in this file?